### DISCUSSION TITLE
Issue turned to pull request stops tracking

### DISCUSSION THREAD

**Discussion by AndreasPB** [community/community#6446] [Projects and Issues]
Hi team,

Our small team got a workflow where we turn small issues into pull requests with the `hub`-cli tool

```bash
git checkout -b <issue-number>/<branch-name>
git add/commit/push
hub pull-request -i <issue-number>
```
This seems to confuse the new Projects board/list which will keep it as an issue and the new pull-request loses the projects-link

**Before:**
![image](https://user-images.githubusercontent.com/43907402/137624787-4e7f482f-f40c-49b8-b915-c61e23f93821.png)
![image](https://user-images.githubusercontent.com/43907402/137624692-4f9673fb-7572-4f17-9630-cdc336eb7a55.png)
**After:**
![image](https://user-images.githubusercontent.com/43907402/137624787-4e7f482f-f40c-49b8-b915-c61e23f93821.png)
![image](https://user-images.githubusercontent.com/43907402/137624697-e7901273-dad9-494c-838c-4a55aac0bdbb.png)

I hope this can get fixed - It's a really promising tool that we will be using a lot! :smiley:

**Comment by iozz** [comment]
Same problem for us.

We create issues, add them to our project, then when we start working on them we convert the issue into a Pull Request using the CLI command `hub pull-request -i <issue number>` ([doc](https://hub.github.com/hub-pull-request.1.html)).

The issue/pull request disappears after it's converted, and we cannot add it back, from the project itself or from the PR. It's systematic and really annoying for us.
